Automattic/wp-calypso
# Odyssey Stats: Apply Emerald typography styling

Fixes #72107.

#### Proposed Changes

|||
|-|-|
|Before|<img width="1792" alt="image" src="https://user-images.githubusercontent.com/4044428/215632462-885d1d25-2aa8-4b79-97ea-9284f0ce774b.png">|
|After|<img width="1792" alt="image" src="https://user-images.githubusercontent.com/4044428/215632325-501bb52c-b398-4adf-9947-77afd039125e.png">|

* Moves a styling mixin from `client/my-sites/stats/` to `packages/components/src/styles/`.
* Introduce a new styling mixin for overriding display type styling in Odyssey Stats.

#### Testing Instructions



* Check the live branch for this PR and ensure that all the type font remains unchanged from production.
* Apply this change to your local Jetpack site's Odyssey Stats (option 1 at PejTkB-3E-p2).
* Ensure that the headings now use a sans-serif font instead of Recoleta. Make sure to check both the traffic and insights pages!
